scripts/extensions/markForUser/src/managed-marked-user.tsx

```diff
@@ -1,13 +1,27 @@
 import {ISuperdesk, IArticle} from 'superdesk-api';
 import {getMarkForUserModal} from './get-mark-for-user-modal';
 
-export function manageMarkedUserForSingleArticle(superdesk: ISuperdesk, article: IArticle) {
+export function manageMarkedUserForSingleArticle(superdesk: ISuperdesk, article: IArticle, sendToNextStage = false) {
+    const task: IArticle['task'] = article.task || null;
+
+    if (sendToNextStage && task?.desk) {
+        superdesk.entities.desk.getDeskStages(task.desk)
+            .then((stages) => {
+                const currentStageIndex = stages.findIndex((stage) => task.stage === stage._id);
```

Comment:
right stages are ordered by creation date. and it doesn't change desk, it moves it from one stage to another on a single desk. stages are created to match the workflow, like jira swimlanes

Reply:
It wouldn't make sense to order jira swimlanes by creation date. If I wanted to create another lane like `Is being tested` and wanted to place it between `review` and `done` it wouldn't work, because in our setup, we can only append stages, not insert new ones in the middle/start.

Reply:
If I remember correctly, users can reorder stages in the monitoring view to their preference. Wouldn't it be better to respect that order? How are they supposed to know which stage is "the next stage" if next is relative to the creation date? I don't believe users remember the order at which the stages were created :smile:

Reply:
it's just how it works now, eventually there should be ordering, but this is a limitation everyone is aware of